# HG changeset patch
# User Rik <rik@octave.org>
# Date 1384303582 28800
#      Tue Nov 12 16:46:22 2013 -0800
# Node ID 88c37cf66fb5fb0d3818747844137b5a2b99cd89
# Parent  7df8f5372ea887cfec5bd3b6bd80c7985b1ebdfd
moment.m: Add @nospell macro around "ar" so spellcheck passes.

* moment.m: Add @nospell macro around "ar" so spellcheck passes.

diff --git a/scripts/statistics/base/moment.m b/scripts/statistics/base/moment.m
--- a/scripts/statistics/base/moment.m
+++ b/scripts/statistics/base/moment.m
@@ -81,17 +81,17 @@
 ## @example
 ## @group
 ## moment (x) = 1/N SUM_i x(i)^p
 ## @end group
 ## @end example
 ##
 ## @end ifnottex
 ##
-## @item @qcode{"ar"}
+## @item @nospell{@qcode{"ar"}}
 ##   Absolute Raw Moment.  The moment about zero ignoring sign defined as
 ## @tex
 ## $$
 ## {\sum_{i=1}^N {\left| x_i \right|}^p \over N}
 ## $$
 ## @end tex
 ## @ifnottex
 ##
